cumulus/parachains/runtimes/assets/asset-hub-rococo/src/xcm_config.rs

```diff
@@ -117,7 +120,7 @@ pub type FungibleTransactor = FungibleAdapter<
 	// Use this currency:
 	Balances,
 	// Use this currency when it is a fungible asset matching the given location or name:
-	IsConcrete<TokenLocation>,
+	(IsConcrete<TokenLocation>, IsConcrete<TokenLocationAsNative>),
```

Comment:
Every parachain in the ecosystem needs to do an equivalent change to their runtime, If we make this equivalence in AssetHub where `RelayLocation` and `AssetHubLocation` are both describing DOT/KSM asset.

Otherwise one parachain could align early with our change, then send DOT to another parachain, and send it through AH by referring to the DOT as `(1, Parachain(AssetHubId))`. AH will happily handle the reserve transfer and send it to the final/destination parachain, but that chain doesn't (yet) recognize `(1, Parachain(AssetHubId))` as being the same asset as `(1, Here)`.

That's why this equivalence needs to be done at the protocol/ecosystem level through https://github.com/polkadot-fellows/RFCs